integration-test-groups/cxf-soap/cxf-soap-server/src/main/java/org/apache/camel/quarkus/component/cxf/soap/server/it/CxfSoapRoutes.java

```diff
@@ -40,6 +40,10 @@ public void configure() {
         from("cxf:bean:soapServiceEndpoint")
                 .setBody().simple("Hello ${body} from CXF service");
 
+        from(String.format("cxf:///hello-uri-address?wsdlURL=wsdl/HelloService.wsdl&serviceClass=%s",
+                HelloPortType.class.getCanonicalName()))
```

Comment:
It does not make any difference for most classes incl. CalculatorService, but generally Class.getName() is the appropriate format here, because Camel is internally passing to ClassLoader.loadClass(String) which expects the Class.getName() format with `$` as a delimiter for inner classes.

```suggestion
                HelloPortType.class.getName()))
```